roles/provision-keycloak-apb/tasks/provision-keycloak.yml

```diff
@@ -36,25 +36,25 @@
         value: '{{ ADMIN_PASSWORD }}'
       - name: PROXY_ADDRESS_FORWARDING
         value: 'true'
-      - name: POSTGRES_USER
+      - name: DB_VENDOR
+        value: postgres
+      - name: DB_ADDR
+        value: '{{ postgres_service_name }}'
+      - name: DB_USER
         value_from:
           secret_key_ref:
             name: '{{ postgres_secret_name }}'
             key: database-user
-      - name: POSTGRES_PASSWORD
+      - name: DB_PASSWORD
         value_from:
           secret_key_ref:
             name: '{{ postgres_secret_name }}'
             key: database-password
-      - name: POSTGRES_DATABASE
+      - name: DB_DATABASE
         value_from:
           secret_key_ref:
             name: '{{ postgres_secret_name }}'
             key: database-name
-      - name: POSTGRES_PORT_5432_TCP_PORT
-        value: "5432"
-      - name: POSTGRES_PORT_5432_TCP_ADDR
-        value: "{{postgresIP.stdout}}"
```

Comment:
We probably don't need [this task](https://github.com/aerogearcatalog/keycloak-apb/blob/2e3d00d3cab321384ca0e4adb5646759a5e1c583/roles/provision-keycloak-apb/tasks/provision-keycloak.yml#L12-L14) to get postgres IP

Reply:
yes, good point. Will remove it.